lib/ace/edit_session/bracket_match.js

```diff
@@ -37,17 +37,84 @@ var Range = require("../range").Range;
 
 function BracketMatch() {
 
+
+    this.findSurroundingBrackets = function(position) {
+    if (position.column == 0) return null;
+
+    var allBracets = { 
```

Comment:
sorry? I meant that you have a typo in the name _allBracets_. I don't mind using _all_ prefix.